examples/new_node_pool.tf

```diff
@@ -19,5 +16,5 @@ module "node_pool_green" {
 }
 
 module "kube_token_2" {
-  source = "modules/kube-token"
+  source = "equinix/multiarch-k8s/metal//modules/kube-token"
```

Comment:
Two //s

Reply:
https://www.terraform.io/language/modules/sources#modules-in-package-sub-directories
This is intentional